### ISSUE TITLE
[Identity] Getting a token via VisualStudioCodeCredential using v0.10.0 of the Azure Account Extension results in an error

### ISSUE THREAD

**Issue by KarishmaGhiya** [Azure/azure-sdk-for-python#23249]
Name of the cache where the token is stored has [changed from version 0.9.11 to 0.10.0 for Azure Account extension for VS Code](https://github.com/microsoft/vscode-azure-account/issues/443#issuecomment-1048018144). As a result of this the Visual Studio Code Credential throws an error/ exception in JS with the following message because the cache with that name is not found
![image](https://user-images.githubusercontent.com/8968058/156138536-2468c28c-21b1-43c9-95c1-221e0913f5bb.png)

JS Bug filed by customer - https://github.com/Azure/azure-sdk-for-js/issues/20500
VS Code Account extension bug filed by customer - https://github.com/microsoft/vscode-azure-account/issues/443

**Comment by lidong-zhao** [comment]
Any workaround for bug? looks like not possible to install previous version of Azure Account.

**Comment by xiangyan99** [comment]
> Any workaround for bug? looks like not possible to install previous version of Azure Account.

In the drop-down list of uninstall, there is an option "install another version".

**Comment by billwert** [comment]
This issue will be addressed in a future releae. #25713

**Comment by billwert** [comment]
Hello @Jan-Kowalik-mulberryrisk!

This hasn't been implemented yet. You should not roll back to a previous version of the extension as that can cause other problems. In the interim the recommendation is to authenticate using the [Azure CLI](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/README.md#authenticate-via-the-azure-cli).

**Comment by Jan-Kowalik-mulberryrisk** [comment]
How do I use `az login` in combination with `launch.json` config? VSCode starts apps, that are configured with it, in a new terminal/shell. 

I tried the below but neither works:
* running `az login` in a terminal and then running the app with `F5`
* running the app with `F5`; killing it; running `az login` in the app terminal and starting the app again with `F5`

Then I tried reverting to the old working version and it works then.
